### ISSUE TITLE
not an issue here, but an open question about supporting your work.

### ISSUE THREAD

**Issue by mihaiaka** [spocky/miproja1#99]
I'm sideloading the GitHub version of your software on my Philips TV (with GoogleTV). Not a fan of the Google Play Store, especially when it comes to software/open source projects/etc like yours. Not a fan of Google Play's commissioning system either.
Do you have or ever considered having an alternative way of getting financial support/donation/etc? Something better (from a commission point of view) like Ko-fi/Patreon/directly to PayPal via link/etc.
P.S. I love this launcher. (might have some later questions/requests, but first I need to figure out how to support your time spent)

BR,
M.

**Comment by cowballz** [comment]
I mentioned the same thing over at his XDA forums. This launcher is worthy of a link to extra funds for the time spent on it

**Comment by truth7seeker** [comment]
I agree I'd love to donate, I bought it off Google, but I'd give and donate more to put it on other devices.

**Comment by spocky** [comment]
Thanks !
Sorry it took so long, I've set up a [ko-fi page](https://ko-fi.com/spocky) a few months ago.
